internal/core/data/v2/application/event.go

```diff
@@ -26,6 +28,12 @@ import (
 	"github.com/google/uuid"
 )
 
+type EventBackgroundScrubber struct {
+	dic                       *di.Container
+	deleteEventsByIdChannel   chan []string
+	deleteReadingsByIdChannel chan []string
```

Comment:
for now, just make these package private variables. I think the App layer should be refactored to be a object with a new method, but that needs to wait.